sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample2_RecognizeReceiptsFromFile.cs

```diff
@@ -32,31 +32,75 @@ public async Task RecognizeReceiptsFromFile()
+                        {
+                            string merchantName = merchantNameField.Value.AsString();
+
+                            Console.WriteLine($"    Merchant Name: '{merchantName}', with confidence {merchantNameField.Confidence}");
+                        }
+                    }
 
-                    string merchantName = usReceipt.MerchantName?.Value ?? default;
-                    DateTime transactionDate = usReceipt.TransactionDate?.Value ?? default;
-                    IReadOnlyList<USReceiptItem> items = usReceipt.Items ?? default;
-                    float subtotal = usReceipt.Subtotal?.Value ?? default;
-                    float tax = usReceipt.Tax?.Value ?? default;
-                    float tip = usReceipt.Tip?.Value ?? default;
-                    float total = usReceipt.Total?.Value ?? default;
+                    FormField transactionDateField;
+                    if (receipt.RecognizedForm.Fields.TryGetValue("TransactionDate", out transactionDateField))
+                    {
+                        if (transactionDateField.Value.Type == FieldValueType.Date)
+                        {
+                            DateTime transactionDate = transactionDateField.Value.AsDate();
 
-                    Console.WriteLine($"Recognized USReceipt fields:");
-                    Console.WriteLine($"    Merchant Name: '{merchantName}', with confidence {usReceipt.MerchantName.Confidence}");
-                    Console.WriteLine($"    Transaction Date: '{transactionDate}', with confidence {usReceipt.TransactionDate.Confidence}");
+                            Console.WriteLine($"    Transaction Date: '{transactionDate}', with confidence {transactionDateField.Confidence}");
+                        }
+                    }
 
-                    for (int i = 0; i < items.Count; i++)
+                    FormField itemsField;
+                    if (receipt.RecognizedForm.Fields.TryGetValue("Items", out itemsField))
                     {
-                        USReceiptItem item = usReceipt.Items[i];
-                        Console.WriteLine($"    Item {i}:  Name: '{item.Name.Value}', Quantity: '{item.Quantity?.Value}', Price: '{item.Price?.Value}'");
-                        Console.WriteLine($"    TotalPrice: '{item.TotalPrice.Value}'");
+                        if (itemsField.Value.Type == FieldValueType.List)
+                        {
+                            foreach (FormField itemField in itemsField.Value.AsList())
+                            {
+                                if (itemField.Value.Type == FieldValueType.Dictionary)
+                                {
+                                    IReadOnlyDictionary<string, FormField> itemFields = itemField.Value.AsDictionary();
+
+                                    FormField itemNameField;
+                                    if (itemFields.TryGetValue("MerchantName", out itemNameField))
```

Comment:
These `items` still correspond to the previously know model `USReceiptItem`  i.e `Quantity`, `Name` ..?

Reply:
Yes.  I only show two of them since the rest of the code would be very repetitive.  The available fields will need to be documented to make them usable -- the service team has requested to own this.

Reply:
How are we thinking about handling the extra/undocumented fields/How do we keep them discoverable?